pkg/apis/operator/v1alpha1/tektonchains_lifecycle.go

```diff
@@ -0,0 +1,183 @@
+/*
+Copyright 2020 The Tekton Authors
```

Comment:
`2020` > `2022`
in all the new files I think  😝

Reply:
handled